.github/workflows/pull_request.yaml

```diff
@@ -30,14 +29,18 @@ jobs:
           --health-retries 5
     steps:
       - uses: actions/checkout@v3
+      - uses: abatilo/actions-poetry@v2.1.5
+        with:
+          poetry-version: 1.2.0
+      - uses: actions/setup-python@v4
+        with:
+          python-version-file: .python-version
+          cache: 'poetry'
```

Comment:
setup poetry in the same way you will do in the container

Reply:
?